archinstall/lib/profiles.py

```diff
@@ -177,7 +177,7 @@ def has_prep_function(self):
 					if hasattr(imported, '_prep_function'):
 						return True
 		return False
-"""
+
```

Comment:
We can leave the comments there to keep the function "out".
But the comments need to be in the correct indentation level: c5d099c

Reply:
I'm fine if you want to make that commit and leave it commented though.

Reply:
Okay, I see you fixed the indentation in that commit in master, I'll drop this from the change.